justice/util.py

```diff
@@ -0,0 +1,29 @@
+"""Utility functions"""
```

Comment:
i prefer slightly more descriptive module names, maybe `light_curves` or `light_curve_base`? util/misc can be kind of an anti-pattern

Reply:
I actually moved the functions elsewhere and got rid of the module.  `util` and `base` modules are just too tempting for me.